swedish-oidc-profile.md

```diff
@@ -349,50 +347,56 @@ Relying Parties MUST follow the requirements in section [3.1.3.7] of \[[OpenID.C
+endpoint.
+
+Relying Parties MUST follow section 5.3.4 of \[[OpenID.Core](#openid-core)\] when validating a
+UserInfo response message.
 
 <a name="claims-release-requirements"></a>
 ### 4.2. Claims Release Requirements
 
 OpenID Providers MUST return claims on a best effort basis. However, an OpenID Provider asserting it can provide a user claim does not imply that this data is available for all its users. Relying Parties MUST be prepared to receive partial data. 
 
-An OpenID Provider compliant with this profile MUST NOT release any identity claims<sup>1</sup> in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via the `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. Furthermore, an OpenID Provider MUST NOT release any claims to a Relying Party that has not been authorized to receive
-them. 
+An OpenID Provider compliant with this profile MUST NOT release any identity claims in the ID token, or via the UserInfo endpoint, if they have not been explicitly requested via `scope` and/or `claims` request parameters, or by a policy known, and accepted, by the involved parties. 
 
-Section 5.4 of \[[OpenID.Core](#openid-core)\] states:
+The above requirement does not include the mandatory `sub` claim, and claims that do not reveal identity
+information about the user, for example, transaction identifiers or claims holding information about
+the authentication process.
 
-> The Claims requested by the `profile`, `email`, `address`, and `phone` scope values are returned from the UserInfo Endpoint, as described in Section 5.3.2, when a `response_type` value is used that results in an Access Token being issued. However, when no Access Token is issued (which is the case for the `response_type` value `id_token`), the resulting Claims are returned in the ID Token. 
+Furthermore, an OpenID Provider MUST NOT release any claims to a Relying Party that has not been authorized to receive them. How this authorization is handled and managed is out of scope for this
+profile. 
 
-This means that, unless explicitly requested via the `claims` request parameter, identity
-claims<sup>1</sup> are delivered from the UserInfo endpoint. 
+An OpenID Provider compliant with this profile MUST adhere to the following claim release requirements:
 
-OpenID Providers compliant with this profile MUST adhere to the above statements, unless governed
-by an overriding policy that states otherwise.
-  
-"Authentication Information Claims" as defined in section 2.3 of \[[OIDC.Sweden.Attr](#attr-spec)\] are
-claims describing an authentication event, and are not regarded as user identity claims. Therefore, any
-of these claims SHOULD be delivered in the ID token and SHOULD NOT be delivered from the UserInfo
-endpoint.
+- If a `claims` request parameter is included in the authentication request for a specific claim,
```

Comment:
This sentence may imply that an OpenID Provider compatible with this profile MUST support the `claims` parameter. 

OpenID Connect Core 1.0 https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter specifies support for the `claims` parameter as OPTIONAL.

Should we support for the `claims` parameter be MANDATORY in this profile?

Reply:
Section 2.1 states: "OPTIONAL for RP and REQUIRED for OP" regarding support of the `claims` parameter.